pypeit/spectrographs/gemini_gmos.py

```diff
@@ -91,9 +91,10 @@ def default_pypeit_par():
         Set default parameters for Keck LRISb reductions.
         """
         par = pypeitpar.PypeItPar()
-        # Set wave tilts order
         par['calibrations']['slits']['sigdetect'] = 20.
```

Comment:
shall we remove all the 'slits' par entries?
seems like it..

Reply:
Sorry to continue to punt on these, but this was another todo for the clean-up PR.